packages/build/src/steps/run_step.ts

```diff
@@ -79,9 +79,15 @@ export const runStep = async function ({
     'build.execution.step.origin': origin,
     'build.execution.step.event': event,
   }
+
+  if (pluginPackageJson?.name && pluginPackageJson?.version) {
+    attributes['build.execution.step.plugin_name'] = pluginPackageJson.name
+    attributes['build.execution.step.plugin_version'] = pluginPackageJson.version
+  }
+
   const spanCtx = setMultiSpanAttributes(attributes)
   // If there's no `coreStepId` then this is a plugin execution
-  const spanName = `run-step-${coreStepId || 'plugin'}`
+  const spanName = `run-step-${coreStepId || `plugin-${event}`}`
```

Comment:
better readability for plugins resulting in a name like `run-step-plugin-onPreBuild` instead of `run-step-plugin` which does not include the build event

Reply:
do we need to check if any of honeycomb dashboards / alerts rely on `run-step-plugin` span name?

Reply:
I have not found something